source/process/bug-fix-release.md

```diff
@@ -106,27 +103,21 @@ Review the [Release Features & Bugs Quality Gate Guidelines](https://docs.google
     - Update Redux before each RC and Final build
     - Master is tagged and branched and `Release Candidate 1` is cut (e.g., 3.5.0-RC1) according to the Release Candidate Checklist in ``mattermost/process``
     - After branching, the database version in `sql_upgrade.go` on master is set to the next scheduled release version (e.g., 3.6.0)
-    - CI servers are updated to the release branch
```

Comment:
What happens to all these removed items?

Edit: Looks like doc tasks were moved to later in the process. Should there be a step where you ask PMs/tech writing to complete docs by a certain date (which you already do, not sure where it's in the process)?